composer.json

```diff
@@ -20,8 +20,11 @@
         }
     ],
     "require" : {
+        "ext-ctype": "*",
         "ext-dom": "*",
+        "ext-iconv": "*",
```

Comment:
iconv is used only as fallback when mbstring is not present, so technically both iconv and mbstring are optional (but at least one of them is required)

Any idea on how to express this in composer?

Reply:
You are right, I don't think there is any way to ask Composer for that. Let's keep an exception :) .